### DISCUSSION TITLE
How to test windows focusing state?

### DISCUSSION THREAD

**Discussion by davidegasperi** [ProjectMirador/mirador#3515] [Q&A]
Hi, I'm a newbie with Redux and can't verify which of the M3 windows I've created with addWindow is currently focused. Are there sufficiently documented APIs on how to interact with MiradorIstance via js? How should I structure the call?
Thanks in advance

**Comment by marlo-longley** [comment]
Hi @davidegasperi we are auditing these stale discussion questions, sorry it was never answered, some new maintainers were not aware of this area. I've converted your question to a ticket for documentation.

**Comment by marlo-longley** [comment]
https://github.com/ProjectMirador/mirador/issues/4517
